### PR TITLE
updated for impact changes

### PR DESCRIPTION
**Pull request by robthew**
Adds new descriptions field and changes impact to a string.

### CODE REVIEW

File: app/views/evaluations/show.html.erb [hunk 1]
```diff
@@ -967,25 +967,29 @@ function draw_status_pie_chart(d) {
...
     for(var i=0, len=d.controls.length; i < len; i++) {
       impact = d.controls[i].impact;
-      if (impact <= 0.4) {
+      if (impact == 'low') {
         low += 1;
-      } else if (impact < 0.7) {
+      } else if (impact == 'medium') {
         medium += 1;
-      } else if (impact >= 0.7) {
+      } else if (impact >= 'high') {
```
Comment (aaronlippold) [inline app/views/evaluations/show.html.erb]: typo I think : `>=` vs `==`

File: app/views/evaluations/show.html.erb [hunk 2]
```diff
@@ -967,25 +967,29 @@ function draw_status_pie_chart(d) {
...
         high += 1;
+      } else if (impact >= 'critical') {
```
Comment (aaronlippold) [inline app/views/evaluations/show.html.erb]: `>=` here?

File: spec/helpers/application_helper_spec.rb [hunk 3]
```diff
@@ -3,9 +3,9 @@
 RSpec.describe ApplicationHelper, type: :helper do
   describe 'category_button' do
     it 'gets a button from an impact' do
-      expect(helper.category_button(0.2)).to eq('btn btn-category-iii')
-      expect(helper.category_button(0.5)).to eq('btn btn-category-ii')
-      expect(helper.category_button(0.8)).to eq('btn btn-category-i')
+      expect(helper.category_button('low')).to eq('btn btn-category-iii')
+      expect(helper.category_button('medium')).to eq('btn btn-category-ii')
+      expect(helper.category_button('high')).to eq('btn btn-category-i')
```
Comment (aaronlippold) [inline spec/helpers/application_helper_spec.rb]: Do we need a test for None/NA and Critical for coverage?

File: spec/models/control_spec.rb [hunk 4]
```diff
@@ -42,17 +42,17 @@
     end
 
     it 'get low severity' do
```
Comment (aaronlippold) [inline spec/models/control_spec.rb]: same as above, do we/should we have ones for 'none'/"Not Applicable" and Critical?

**Comment by samcornwell** [comment]
@robthew @aaronlippold I'll go ahead and make changes to store numerical impacts in the database, move conversion to display purposes
